src/components/PageHeader/extras/Extras.tsx

```diff
@@ -0,0 +1,15 @@
+'use client'
+
+import React, { ReactElement } from 'react'
+
+import { useFlag } from '../../../toggles/context'
+
+import Flexjar from './Flexjar'
+
+function Extras(): ReactElement {
+    const flexjarToggle = useFlag('SYK_DIG_FLEXJAR_HEADER')
+
+    return <div className="flex gap-3 items-center">{flexjarToggle.enabled && <Flexjar />}</div>
```

Comment:
```suggestion
    if (!flexjarToggle.enabled) return null
```

Eller så blir det ev. berre ein tom div?